### DISCUSSION TITLE
Color scheme (by value) thresholds per column not per table

### DISCUSSION THREAD

**Discussion by ant-mermaid-street** [grafana/grafana#58824] [Dashboards, Panels & Visualizations]
Hello
I will provide a basic example for a better understanding of this feature request.
I run two queries returning two sets of results with a identical common field. I Outer join by that field to obtain this table output:
(I am not sure if relevant or not how the table is generated, I suppose it can work with single queries as well)

![image](https://user-images.githubusercontent.com/22320901/202182841-802cc3e4-d0f8-4ef0-9a64-d807fc1c774b.png)

the mix and max per column are very different
0.06 to 1.4 on the first column
1.93 to 840 on the second column

I then apply a cell display mode override "Gradient Gauge"  and a color scheme "green-yellow-red" by value and I obtain the following results:

![image](https://user-images.githubusercontent.com/22320901/202183377-057a48ad-9132-428b-8f89-27266e270e59.png)
![image](https://user-images.githubusercontent.com/22320901/202183773-5648992c-40b5-445b-bfce-8cf52005cff8.png)

Judging by how the colors are displayed it looks like the color scheme and the gradient gauge will take into consideration the min and the max for all the values in the table. from 0.06 to 840
While this is useful in certain situations, we should also have the possibility to chose and have these thresholds per column instead of per table as it  is configured by default.

Currently I am using a workaround which implies the definition of manual thresholds values/colors and then setting static max/min limits for each column. This is not ideal when working with dashboards based on variables as the thresholds values will not apply to other similar tables in different context.

Thank you for looking at this feature request.

**Comment by dp106** [comment]
I definitely need this feature as it's current behaviour makes setting a % threshold useless to me when creating dashboards. Case in point my latest dashboard combines the size of a database with the reads per second and writes per second.  The reads and writes are always green because the values for database size are so much larger.

**Comment by haeraeus** [comment]
Is this a duplicate of https://github.com/grafana/grafana/issues/26383?

**Comment by dp106** [comment]
Yes it does appear to be the same issue. No sign of it being fixed though 😥

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move [here](https://github.com/grafana/grafana/discussions/73424).

I will be closing this discussion, but if you’re still interested in this request being implemented please don’t hesitate to open a [feature request](https://github.com/grafana/grafana/issues/new?assignees=&labels=&projects=&template=1-feature_requests.md). Sorry for the inconvenience, and thanks for your contributions!

cc @grafana/grafana-bi-squad
